AmplifyPlugins/Auth/AWSCognitoAuthPluginTests/AuthenticationProviderTests/AuthenticationProviderConfirmSignupTests.swift

```diff
@@ -26,7 +26,7 @@ class AuthenticationProviderConfirmSignupTests: BaseAuthenticationProviderTest {
     ///
     func testSuccessfulConfirmSignUp() {
 
-        let mockSignupResult = SignUpResult(signUpState: .confirmed, codeDeliveryDetails: nil)
+        let mockSignupResult = SignUpResult(signUpState: .confirmed, codeDeliveryDetails: nil, userSub: nil)
```

Comment:
what's this from?

Reply:
I think this is from the modle update of AWSCognito